api/src/main/java/jakarta/persistence/PessimisticLockScope.java

```diff
@@ -11,53 +11,95 @@
  */
 
 // Contributors:
+//     Gavin King      - 4.0
 //     Linda DeMichiel - 2.1
 //     Linda DeMichiel - 2.0
 
 package jakarta.persistence;
 
 /**
- *
- * Defines the values of the {@code jakarta.persistence.lock.scope}
- * property for pessimistic locking.  This property may be passed as an
- * argument to the methods of the {@link EntityManager}, {@link Query},
- * and {@link TypedQuery} interfaces that allow lock modes to be specified
- * or used with the {@link NamedQuery} annotation.
+ * Controls how a {@linkplain LockModeType pessimistic lock} applied to
+ * an entity affects associated collections and relationships.
+ * <p>
+ * A {@code PessimisticLockScope} may be passed as a {@link FindOption}
+ * {@link RefreshOption}, or {@link LockOption}. Alternatively, it may
+ * be specified via the property {@code jakarta.persistence.lock.scope}.
+ * This property may be passed as an argument to methods of
+ * {@link EntityManager}, {@link Query}, and {@link TypedQuery} which
+ * accept a {@link LockModeType} or it may be used in the {@link NamedQuery}
```

Comment:
Oh I suppose you want this to say:

```suggestion
 * This property may be passed as an argument to methods of
 * {@link EntityManager} which accept a {@link LockModeType} or to 
 * {@link Query#setHint}, or it may be used in the {@link NamedQuery}
```